src/sentry/profiles/task.py

```diff
@@ -773,28 +773,31 @@ def on_symbolicator_request():
                 ],
                 platform=profile["platform"],
             )
-            if not response:
-                profile["symbolicator_error"] = {
-                    "type": EventError.NATIVE_INTERNAL_FAILURE,
-                }
-            elif response["status"] == "failed":
-                profile["symbolicator_error"] = {
-                    "type": EventError.NATIVE_SYMBOLICATOR_FAILED,
-                    "status": response["status"],
-                    "message": response["message"],
-                }
-            elif len(response["errors"]) > 0:
-                profile["symbolicator_error"] = response["errors"][0]
-            elif len(response["stacktraces"]) > 0:
-                merge_jvm_frames_with_android_methods(
-                    frames=response["stacktraces"][0]["frames"],
-                    methods=profile["profile"]["methods"],
-                )
-                return True
+            if response:
+                if response["status"] == "failed":
+                    sentry_sdk.set_context(
+                        "deobfuscation_error",
+                        {
+                            "status": response["status"],
+                            "message": response["message"],
+                        },
+                    )
+                    sentry_sdk.capture_message("Deobfuscation via Symbolicator failed")
+                elif "stacktraces" in response:
+                    if "errors" in response:
+                        sentry_sdk.set_context(
+                            "deobfuscation_error",
+                            {
+                                "errors": response["errors"],
+                            },
+                        )
```

Comment:
is this meant to be nested in the successful case?

Reply:
It's not. From what I understand, we might have an error for a given ProGuard file (it's missing, we can't fetch it from GCS, etc) but the response will still be successful, just incomplete.